test/smoke/src/areas/positron/new-project-wizard/new-project.test.ts

```diff
@@ -38,7 +32,7 @@ export function setup(logger: Logger) {
 				});
 				it('Create a new Conda environment [C628628]', async function () {
 					// This test relies on Conda already being installed on the machine
-					const projSuffix = '_condaInstalled';
+					const projSuffix = '_condaInstalled_' + Math.floor(Math.random() * 1000000).toString();
```

Comment:
this test has been flakey. and on retry it can't create a new project bc project name already exists - so i appended a random # at end to avoid that conflict.